crates/viewer/re_redap_browser/src/servers.rs

```diff
@@ -115,6 +117,8 @@ impl Server {
         ui: &mut egui::Ui,
         dataset: &Dataset,
     ) {
+        const RECORDING_LINK_FIELD_NAME: &str = "recording link";
```

Comment:
What is this hard-coded string? Where does it come from?

Reply:
This is a column that is live injected using datafusion.

Reply:
Where is it live-injected? Who gives it this name? What other piece of code needs to be updated in tandem with this piece of code?

Reply:
As per the location of this const suggests, this is all local to this method. `generate_partition_links` is what triggers that column generation (it takes a column name as input), and `default_column_visibility` defines, well, what the default column visibility (and it wants that generated column to be visible by default).